test/integration/test_factual_knowledge.py

```diff
@@ -19,7 +19,7 @@
 ABS_TOL = 1e-4
 os.environ["PARALLELIZATION_FACTOR"] = "2"
 
-config = FactualKnowledgeConfig("<OR>")
+config = FactualKnowledgeConfig("<OR>", "OR")
```

Comment:
If you do not change this config, the exist tests should also pass, right?

Reply:
yup! I just wanted to add "OR" becuase "<OR>" was also there